File: java/task4.java
import java.util.*;

public class task4 {

    static class Person {
        final String first;
        final String last;
        final int age;
        Person(String first, String last, int age) {
            this.first = first; this.last = last; this.age = age;
        }
        @Override public String toString() {
            return last + ", " + first + " (" + age + ")";
        }
    }

    public static void main(String[] args) {
        Person[] people = new Person[] {
            new Person("Ben", "Adams", 30),
            new Person("Ana", "Zhang", 22),
            new Person("Cara", "Lopez", 27),
            new Person("Ben", "Adams", 35)
        };


        for (Person p : people) System.out.println(p);
    }
}

File: python/task4.py
# orders_data.py
from typing import List, Dict, Any

Order = Dict[str, Any]
ORDERS: List[Order] = [
    {"id": 101, "total": 29.99, "created": "2025-01-15T09:30:00", "customer": {"first": "Ana", "last": "Zhang"}},
    {"id": 102, "total": 120.00, "created": "2025-01-14T16:10:00", "customer": {"first": "Ben", "last": "Adams"}},
    {"id": 103, "total": 120.00, "created": "2025-01-14T08:05:00", "customer": {"first": "Ben", "last": "Adams"}},
    {"id": 104, "total": 75.50, "created": "2025-01-13T12:00:00", "customer": {"first": "Cara", "last": "Lopez"}},
]